.gitignore

```diff
@@ -7,6 +7,7 @@ vendor
 
 .phpunit.result.cache
 .php-cs-fixer.cache
+.php_cs.cache
```

Comment:
Should we have one of these files? Isn't it from an old version of PHP CS Fixer?